docs/advanced/unix-roundup.md

```diff
@@ -105,9 +105,15 @@ To enhance the security of your SSH sessions you can run:
 
 ```bash
 geo lock
+# Once you want to stop tracking
+geo stop
 ```
 
-This will disconnect all active SSH connections if the device is moved.
+This will disconnect all active SSH connections if the device is moved. If you only want to enable background tracking for background connections, do:
```

Comment:
It is better to say `geo track` again here. And add significantly after `is moved`